addons/background-blur/addon.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Blur background",
```

Comment:
Name should be changed to "Blur editor background on popup"
```suggestion
  "name": "Blur background",
  "name": "Blur editor background on popup"
```

Reply:
I'll need more opinions on this

Reply:
```suggestion
  "name": "Blur editor popup background",
```

Reply:
> 

Ngl, that will mean blurring the popup's background.

Reply:
`editor-dark-mode` calls it a popup backdrop.